statsmodels/tsa/tests/test_arima_process.py

```diff
@@ -127,12 +137,189 @@ def test_armafft():
             assert_almost_equal(ac1, ac2, decimal=7,
                                 err_msg='acovf not equal for %s, %s' % (ar, ma))
 
+def test_lpol2index_index2lpol():
+    process = ArmaProcess([1, 0, 0, -0.8])
+    coefs, locs = lpol2index(process.arcoefs)
+    assert_almost_equal(coefs, [0.8])
+    assert_equal(locs, [2])
+
+    process = ArmaProcess([1, .1, .1, -0.8])
+    coefs, locs = lpol2index(process.arcoefs)
+    assert_almost_equal(coefs, [-.1, -.1, 0.8])
+    assert_equal(locs, [0, 1, 2])
+    ar = index2lpol(coefs, locs)
+    assert_equal(process.arcoefs, ar)
+
+
+class TestArmaProcess(TestCase):
+    def test_empty_coeff(self):
+        process = ArmaProcess()
+        assert_equal(process.arcoefs, np.array([]))
+        assert_equal(process.macoefs, np.array([]))
+
+        process = ArmaProcess([1, -0.8])
+        assert_equal(process.arcoefs, np.array([0.8]))
+        assert_equal(process.macoefs, np.array([]))
+
+        process = ArmaProcess(ma=[1, -0.8])
+        assert_equal(process.arcoefs, np.array([]))
+        assert_equal(process.macoefs, np.array([-0.8]))
+
+    def test_from_coeff(self):
+        ar = [1.8, -0.9]
+        ma = [0.3]
+        process = ArmaProcess.from_coeffs(np.array(ar), np.array(ma))
+
+        ar.insert(0, -1)
+        ma.insert(0, -1)
```

Comment:
@bashtage Did you intentionally use a negative version of the lag polynomials here?

I'm not sure what it means for the ARMA process except that the filtered series has reversed sign, and the asserts pass.